scripts/sv_logs.lua

```diff
@@ -0,0 +1,111 @@
+if GetConvar('txAdminServerMode', 'false') == 'true' then
+	-- micro optimization
+	local os_time = os.time
+	-- http://lua-users.org/wiki/SimpleRound
+	local function round(num)
+		return tonumber(string.format("%.2f", num))
+	end
+
+	local function getPlayerData(src)
+		if not src then return false end
+		if src == -1 then return {name = 'console', identifiers = {}} end
+		return {
+			name = GetPlayerName(src),
+			identifiers = GetPlayerIdentifiers(src)
+		}
+	end
+
+	local apiPort = GetConvar("txAdmin-apiPort", "invalid")
+	local apiToken = GetConvar("txAdmin-apiToken", "invalid")
+	if apiPort == "invalid" or apiToken == "invalid" then
+		logError('API Port and Token ConVars not found. Do not start this resource if not using txAdmin.')
```

Comment:
I don't think logError was declared, does it carry from the other file? 🤔

Reply:
Yes, its something that carrys over from sv_main, didn't realize that log does too 😨